bin/build-content

```diff
@@ -46,7 +43,11 @@ categories.each do |category, url|
     next if header == "Remark"
 
     file_name = header.gsub(" ", "_").downcase
-    File.open("#{category_path}/#{file_name}.txt", "w") do |file|
+
+    # This is actually called WeightedMethodCount in the UI and code
+    file_name = "weightedmethodcount" if file_name == "excessiveclasscomplexity"
```

Comment:
It might feel like overkill, but we probably want a mapping for this. I'm sure there will be more in the future.